Sources/KlaviyoSwift/Klaviyo.swift

```diff
@@ -564,14 +564,6 @@ public class Klaviyo : NSObject {
         }
     }
     
-    private func updateNetworkActivityIndicator(on: Bool) {
-        if showNetworkActivityIndicator {
-            DispatchQueue.main.async {
-                UIApplication.shared.isNetworkActivityIndicatorVisible = on
-            }
-        }
-    }
-    
```

Comment:
why isnt this needed anymore?

Reply:
It's deprecated and also I think this was a bad idea to mess with this in the first place.